packages/api/test/e2e/api.create.e2e.ts

```diff
@@ -31,13 +30,9 @@ describe('e2e api create', () => {
 
   afterEach(async () => {
     try {
-      if (api) {
+      if (api && api.connected) {
```

Comment:
nit: can you use same as above change: `api.isConnected`?

Reply:
ya.. I am just trying different ways to settle for connection between node running in docker.. I will clear/clean these once I have the connection.. and then make it ready for review.. was just trying to test.. later on searched on how to  run docker compose locally..